<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 208, 255);</string>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Книга успешно добавлена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
